android/BOINC/app/src/main/java/edu/berkeley/boinc/ProjectDetailsFragment.java

```diff
@@ -112,9 +113,14 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
 	@Override
 	public void onAttach(Activity activity) {
 		// first time fragment can get a valid context (before this, getActivity() will return null!)
-		Display display = activity.getWindowManager().getDefaultDisplay(); 
-		width = display.getWidth();
-		height = display.getHeight();
+		Display display = activity.getWindowManager().getDefaultDisplay();
```

Comment:
This line didn't change it's functional content so I presume the indentation changed from `tab` to `space` or vice versa? If that's correct, did you ensure the indentation style of the two remaining lines (115, 124 in your version) are identical?

Reply:
I checked it's the same.